p2p/discover/udp.go

```diff
@@ -153,6 +153,10 @@ func (t *udp) nodeFromRPC(sender *net.UDPAddr, rn rpcNode) (*Node, error) {
 	if t.netrestrict != nil && !t.netrestrict.Contains(rn.IP) {
 		return nil, errors.New("not contained in netrestrict whitelist")
 	}
+	if t.blacklist != nil && t.blacklist.Contains(rn.IP) {
+		log.Proto("BLACKLIST", "addr", rn.IP.String(), "transport", "udp")
+		return nil, errors.New("contained in blacklist")
```

Comment:
can you use `errBlacklisted` here, or does including it cause a circular dependency?

Reply:
actually, jk - this is a different blacklisting mechanism for UDP. the other one is for TCP. this is fine